modules/soe_paragraph_stories/templates/stories-paragraph.ui_patterns.yml

```diff
@@ -1,34 +1,38 @@
-stories-paragraph:
-  label: Stories Paragraph
-  description: Stories Paragraph
-  variants:
-    default:
-      label: Minimal
-      description: "Minimal"
-  fields:
-    image:
-      type: text
-      label: "Image"
-      description: "The image to display for the Paragraph"
-    person_content:
-      type: text
-      label: "Content"
-      description: "The main content area"
-    quote:
-      type: text
-      label: "Quote"
-      description: "The quote for the story"
-    link:
-      type: text
-      label: "Link"
-      description: "Link to the story node"
-    cta:
-      type: text
-      label: "CTA"
-      description: "An optional CTA button"
-  use: "@soe_paragraph_stories/stories-paragraph.html.twig"
-  libraries:
-    - stories-paragraph:
-        css:
-          component:
-            ../../../dist/css/stories-paragraph.styles.css: {}
+stories-paragraph:
+  label: Stories Paragraph
+  description: Stories Paragraph
+  variants:
+    default:
+      label: Minimal
+      description: "Minimal"
+  fields:
+    image:
+      type: text
+      label: "Image"
+      description: "The image to display for the Paragraph"
+    person_content:
+      type: text
+      label: "Content"
+      description: "The main content area"
+    quote:
+      type: text
+      label: "Quote"
+      description: "The quote for the story"
+    link:
+      type: text
+      label: "Link"
+      description: "Link to the story node"
+    cta:
+      type: text
+      label: "CTA"
+      description: "An optional CTA button"
+    border_color_variant:
+      type: integer
+      label: "Border color option"
+      description: "A numerical variant item"
```

Comment:
New field. Probably a better way to do this but good enough for now.

Reply:
I thought about that, but also remembered we were trying to reduce fields in the patterns, so since it was related to the image, I figured that was the right place to pull it in.  I like this solution much better.

Reply:
This should probably be a setting or something a bit fancier but good enough for now.